# Course link

Join Zoom Meeting
https://zoom.us/j/97331013015?pwd=LzliTE1oeDNpbE1HRnBrTlZYVHd0dz09
Meeting ID: 973 3101 3015
Passcode: 454256

# Yeast Overview

![The-sub-cellular-compartmentalization-and-architecture-of-the-budding-yeast-Saccharomyces](.\images\The-sub-cellular-compartmentalization-and-architecture-of-the-budding-yeast-Saccharomyces.png)

# Plasmid

<img src=".\images\plas.jpg" alt="plas" style="zoom:80%;" align="left" />

 **Shuttle vector**

<img src=".\images\shuttle vector.jpg" alt="shuttle vector" style="zoom:80%;" align="left" />

# Bacteria transformation

1.creating small pores in the cell wall and cell membrane(**transient pore** in the membrane)

<img src=".\images\pore.jpg" alt="pore" style="zoom:80%;" align="left" />

2.after the transformation we immediately add SOC media

<img src=".\images\soc.jpg" alt="soc" style="zoom:100%;" align="left" />

3. we have **to give time to bacteria to translate the antibiotic resistance proteins**(ampicillin), therefore, we incubate it 30min-1h in the incubator.

<img src=".\images\amp.jpg" alt="amp" style="zoom:80%;" align="left"/>

4.then spread the cells and incubate overnight.



# Selectable markers

**URA3**

URA3 encodes [Orotidine 5'-phosphate decarboxylase](https://en.wikipedia.org/wiki/Orotidine_5'-phosphate_decarboxylase) **(ODCase)**, which is an [enzyme](https://en.wikipedia.org/wiki/Enzyme) that catalyzes one reaction in the synthesis of [pyrimidine](https://en.wikipedia.org/wiki/Pyrimidine) [ribonucleotides](https://en.wikipedia.org/wiki/Ribonucleotides) (a component of [RNA](https://en.wikipedia.org/wiki/RNA)). 

Loss of ODCase activity leads to a lack of cell growth unless [uracil](https://en.wikipedia.org/wiki/Uracil) or [uridine](https://en.wikipedia.org/wiki/Uridine) is added to the media. The presence of the URA3 gene in yeast restores ODCase activity, facilitating growth on media not supplemented with uracil or uridine, thereby allowing selection for yeast carrying the gene. In contrast, if **[5-FOA (5-Fluoroorotic acid)](https://en.wikipedia.org/wiki/5-Fluoroorotic_Acid)** is added to the media, the active ODCase will convert 5-FOA into the toxic compound (a [suicide inhibitor](https://en.wikipedia.org/wiki/Suicide_inhibitor)) [5-fluorouracil](https://en.wikipedia.org/wiki/5-fluorouracil) causing cell death, which allows for selection against yeast carrying the gene.

Since URA3 allows for both positive and negative selection, it has been developed as a [genetic marker](https://en.wikipedia.org/wiki/Genetic_marker) for DNA transformations and other genetic techniques in bacteria and many fungal species. It is one of the most important genetic markers in [yeast](https://en.wikipedia.org/wiki/Yeast) genetic modification. While URA3 is a powerful selectable marker it has a high background. This background is because cells that pick up mutations in URA3 may also grow on 5-FOA. Colonies should be verified by a second assay such as PCR to confirm the desired strain has been created.



| Yeast selection marker | What does the marker help synthesize? | Counterselection?(growth-based positive selection for the loss of the marker gene) | For use in:     | Can this be used in other yeast species?                     | Can this be used for auxotrophic selection in E. coli? | Additional considerations                                    |
| ---------------------- | ------------------------------------- | ------------------------------------------------------------ | --------------- | ------------------------------------------------------------ | ------------------------------------------------------ | ------------------------------------------------------------ |
| *HIS3*                 | L-hisitidine                          | no                                                           | *S. cerevisiae* | no                                                           | yes                                                    |                                                              |
| *URA3*                 | pyrimidine (uracil)                   | yes - Grow with 5-FOA.                                       | *S. cerevisiae* | yes - This can complement *ura4-* *S. pombe*, but the complementation is weak. | yes                                                    |                                                              |
| *LYS2*                 | L-lysine                              | yes - Grow on plates containing alpha-aminoadipate in the absence of a nitrogen source. | *S. cerevisiae* | no                                                           | no                                                     |                                                              |
| *LEU2*                 | L-leucine                             | no                                                           | *S. cerevisiae* | yes - This can complement leu1- *S. pombe*, but mulitple copies are required. | yes                                                    |                                                              |
| *TRP1*                 | L-tryptophan                          | yes - Grow with 5-FAA.                                       | *S. cerevisiae* | no                                                           | no                                                     | *TRP1* alters some yeast phenotypes. This marker should not be used in gene disruption experiments. |

 **Auxotrophy is defined as the inability of an organism to synthesize a particular organic compound required for its growth.** Many auxotrophic strains of yeast exist which can be easily maintained when grown on media containing the missing nutrient. Scientists can exploit these host mutations by including a copy of a functional gene which complements the host’s auxotrophy. When grown on media NOT containing the nutrient, **the host cells will die unless they have incorporated the plasmid carrying the required gene.** 

# Plasmid Isolation from yeast

<img src=".\images\stet.jpg" alt="stet" style="zoom:100%;" align="left" />

# Miniprep

<img src=".\images\miniprep.jpg" alt="miniprep" style="zoom:100%;" align="left" />

<img src=".\images\miniprepPeg.jpg" alt="miniprepPeg" style="zoom:100%;" align="left" />

# Screening 



<img src=".\images\screening-16296481576125.gif" alt="screening" style="zoom:120%;" />



# Mutagenesis PCR

![mutagenesis PCR](.\images\mutagenesis PCR.jpg)

Fig. 1. Mutagenesis strategies for localized and whole-gene mutagenesis. Error prone PCR is depicted schematically in panel (A) as a strategy for the undirected distribution of mutations over a whole gene of interest. Two types of mutations are shown, favourable ones (open squares) and unfavourable ones (open circles). In successive cycles of PCR, more mutations of each are introduced, and usually molecules will contain some of either type. Thus, the beneficial effect of the favourable mutations can be completely obscured by the presence of unfavourable ones. A possible solution to this is shown in panel (B) and is called DNA shuffling [10]. The PCR product is cut into small fragments by the enzyme DNase I and subsequently reassembled by PCR. Mutations are thereby crossed, and genes with mostly favourable mutations can be enriched by selection. Panel (C) shows the principle of PCR using a 'degenerate' primer. This primer contains a mixture of all four possible nucleotides (abbreviated by the letter 'N') at a certain position or, alternatively, a mixture of codons assembled from trinucleotides [11]. The proteins synthesized from this gene will thus carry a randomised set of amino acids at this position.

<img src=".\images\Error-Prone-PCR-Kit.png" alt="Error-Prone-PCR-Kit" style="zoom:50%;" />



# Bacteria only take up one plasmid?

which mechanism allows only one plasmid copy to get inside the cell during a heat shock transformation?

https://www.researchgate.net/post/Does_anybody_know_which_mechanism_allows_only_one_plasmid_copy_to_get_inside_the_cell_during_a_heat_shock_transformation

**Answer1:** There is no mechanism controlling how many molecules of plasmid get into the cell, but the lower the transformation efficiency the higher the chance that a single cell will only get one molecule.

Ciaran's explanation of why cells can't continue to maintain two plasmids with the same vector but different inserts is correct. Because the cell can't tell the replication controls apart, genetic drift leads to only one version persisting.

**Answer2:** I am in agreement with Rosie's answers: the reason why a bacterial cell appears to only transform with one clone (vector with a unique insert) is due to chance - the lower the clone : bacteria ratio the less likelyhood is their that a single cell would encounter two clone DNA molecules at the same time and transform with both. Rosie's description of the mechanism by which **cells do not retain two similar plasmids is based on the old and well-understood mechanism of incompatibility.** Plasmids have one or more origins of replication, copy-number control determinants, active and passive partition systems and conjugation apparatus. **If two plasmids have the same system, the components of that system (usually DNA repeat elements and proteins, but also RNA) can't distinguish between the two plasmids, and over time (several generations) one or the other plasmid is lost by chance.**

# Esterification

<img src=".\images\wps56.png" alt="img" style="zoom:80%;" align="left" />

In organic chemistry, transesterification is the process of exchanging the organic group R″ of an ester with the organic group R′ of an alcohol. These reactions are often catalyzed by the addition of an acid or base catalyst.

![img](.\images\wps57.png) 

# ACT1 gene+Cup1

ACT1

(Exon1)**CTT TTAGATTTTT CACGCTTACT 961 GCTTTTTTCT TCCCAAGATC GAAAATTTAC TGAATTAACA ATGGATTCTG** GTATGTTCTA 1021 GCGCTTGCAC CATCCCATTT AACTGTAAGA AGAATTGCAC GGTCCCAATT GCTCGAGAGA 1081 TTTCTCTTTT ACCTTTTTTT ACTATTTTTC ACTCTCCCAT AACCTCCTAT ATTGACTGAT 1141 CTGTAATAAC CACGATATTA TTGGAATAAA TAGGGGCTTG AAATTTGGAA AAAAAAAAAA 1201 AACTGAAATA TTTTCGTGAT AAGTGATAGT GATATTCTTC TTTTATTTGC TACTGTTACT 1261 AAGTCTCATG TACTAACATC GATTGCTTCA TTCTTTTTGT TGCTATATTA TATGTTTAG**A 1321 GGTTGCTGCT TTGGTTATTG ATAACGGTTC TGGTATGTGT AAAGCCGGTT**(Exon2) 

+CUP1  

ATG(not include atg) **TTCAGCGAATTAATTAACTTCCAAAATGAAGGTCATGAGTGCCAATGCCAATGTGGTAGCTGCAAAAATAATGAACAATGCCAAAAATCATGTAGCTGCCCAACGGGGTGTAACAGCGACGACAAATGCCCCTGCGGTAACAAGTCTGAAGAAACCAAGA** ......

(Exon2) AAGACACCAAGGTATCATGGTCGGTATGGGTCAAAAA

+LEU(selectable marker)

# ACT1-CUP1 reporter assay

![IMG-2022](.\images\IMG-2022.jpg)



![reporter](.\images\reporter.jpg)

![act1cup1](.\images\act1cup1.jpg)

ACT1 -----**Essential gene**; conditional mutants show pleiotropic defects including abnormal actin cytoskeleton, chromosome instability, altered subcellular morphology, **increased sensitivity to heat, multiple chemicals and osmotic stress; overexpression leads to growth arrest**

***\*Cup1\****-----encodes **Copper metallothionein 1** protein; binds copper and mediates resistance to high concentrations of copper and cadmium; 

**Protects the cell against copper toxicity by tightly chelating copper ions.**

The reporter gene fusion to monitor mRNA splicing in yeast. An intron-containing fragment from the Saccharomyces cerevisiae ***\*ACT1 gene has been fused to CUP1\****, the yeast metallothionein homolog. ***\*CUP1 is a nonessential gene that allows cells to grow in the presence of copper in a dosage-dependent manner.\**** By inserting previously characterized intron mutations into the fusion construct, we have established that the efficiency of splicing correlates with the level of copper resistance of these strains.

![img](.\images\wps58.png) 

A42c+C43a is the upstream of the ACAGA,it’s mutation improves the transition from the first step to the second step.

Prp8 alleles also improve the transition from the first step to the second step. 

Combination of these to effects improves more transition to the second step. 

Once we have two effects they are additive, it means they affect different interactions, if they affect the same interaction, their effects are same instead of being additive.

***\*STET Lysis\**** Solution is a commonly used buffer for the lysis of bacterial cells such as E. ... The buffer contains Sucrose, Triton X, EDTA and Tris which help in maintaining the osmotic pressure, cleaving the cell wall, chelating agent and buffering of the solution respectively in the plasmid isolation procedure.

# cold sensitive markers

PRP16-302 is a ***\*cold sensitive\**** mutant which means yeast cells expressed this protein cannot grow low temperature.

![img](.\images\wps59.png) 

# 5FOA

In the 5FOA plate the URA expressed yeast cells die.

![img](.\images\wps60.png) 

Because

![img](.\images\wps61.jpg) 

 # Recording

***\*transesterification\**** is the process of exchanging the organic group R″ of an ester with the organic group R′ of an alcohol. 

ATPase: prp2,prp16,prp22

ATPase pushing the elements in the splicesome to move.

Mutant prp2 ATPase cannot push the substrate into the first step.

Just early after the transcription, U1 SnRNP binding with 5’SS then U2 SnRNP binds with BS and then these two interacts with each other---this is so called complex A.  then U1 is replaced by U6, U6 is coming in as part of triSnRNP(U4,U5,U6), it’s then U1 is expelled. U4 sits on 5’SS then U4 is released. Then U6 changes its conformation and then the ISL forms. Then later in the spliceosome we have this interaction that we look at them. 

U6 role is stabilizes the 5’SS at the correct position. 

ACAGA region is U6 paired with 5’SS of the intron. This interaction is important for bringing splice site for catalysis. If the interaction is disrupted, the intron less stably bound, therefore, it’s hard to kept it in place for the first step so the first step is inhibited.

When Prp16 is mutated, it’s hard to make the transition, but what can help this defective Prp16?  anything destabilizes the catalytic conformation would help to move things out.  

In the situation when we introduce ***\*mutant Prp16 the first step of the catalysis is improved\**** but the overall splicing is inhibited, because they don’t move to the second step.

This Prp16 mutant makes the cells are alive but they grow only 30 degree. At low temperature they cannot grow.

in general, any mutation that improves growth of cold sensitive Prp16 mutants, we expect if it disrupts the catalytic conformation of the first step. 

 Mutation of upstream of ACAGA destabilize some catalytic interaction of the first step, therefore it promotes transition to the second step.

There are some important interactions that occuring in U6,Prp8,Cwc2 which coincide in one physical region---suggesting that these all three interact with each other and this interaction needs to be disrupted in order to transition to the second step.

When you have two effects they are additive, it means they are different. If they are the same, they can’t help what is already in helped.

**1.** ***\*if improve pairing with ACGA---the first step of the catalysis is improved\****

**2.** ***\*Independent mutation of Prp8,cwc2,U6 upstream ---makes first step worse.\****

If we do the combination of improving the first step in one way then improving the second step in different way not by destroying the first step but by stabilizing the second step then BS-C is so much better.

The BS-C inhibit 1st and 2nd step of catalysis, this will tell us if they are additive.

# Alternative splicing

![DNA_alternative_splicing](E:\NotesTypora\images\DNA_alternative_splicing.gif)

**Alternative splicing**, or **alternative RNA splicing**, or **differential splicing**, is an alternative [splicing](https://en.wikipedia.org/wiki/RNA_splicing) process during [gene expression](https://en.wikipedia.org/wiki/Gene_expression) **that allows a single [gene](https://en.wikipedia.org/wiki/Gene) to [code](https://en.wikipedia.org/wiki/Genetic_code) for multiple [proteins](https://en.wikipedia.org/wiki/Proteins).** In this process, particular [exons](https://en.wikipedia.org/wiki/Exons) of a gene may be included within or excluded from the final, processed [messenger RNA](https://en.wikipedia.org/wiki/Messenger_RNA) (mRNA) produced from that gene.[[1\]](https://en.wikipedia.org/wiki/Alternative_splicing#cite_note-Black-1) This means the [exons](https://en.wikipedia.org/wiki/Exon) are joined in different combinations, leading to different (alternative) mRNA strands. Consequently, the proteins [translated](https://en.wikipedia.org/wiki/Translation_(biology)) from alternatively [spliced mRNAs](https://en.wikipedia.org/wiki/RNA_splicing) will contain differences in their amino acid sequence and, often, in their biological functions (see Figure). Notably, alternative splicing allows the [human genome](https://en.wikipedia.org/wiki/Human_genome) to direct the synthesis of many more proteins than would be expected from its 20,000 protein-coding genes.



# Role of lithium acetate in yeast transformation?

Both lithium acetate and heat shock, which **enhance the transformation efficiency of intact cells** but not that of spheroplasts, probably help DNA to pass through the cell wall.

# Role of glycerol in preparation of competent cells?

add glycerol to the stock solution as cryoprotectant agent(A cryoprotectant is a substance used to protect biological tissue from freezing damage.). Upon freezing the cells face osmotic stress. When glycerol is added to the solution, it will enter the cell membrane and protect the cells from shrinking.

# Ribose 5-phosphate+Base

<img src=".\images\basepair_labeled.gif" alt="basepair_labeled" style="zoom:100%;" align="left" />

<img src=".\images\53.png" alt="53" style="zoom:50%;" align="left" />

# Recombination

recombination--the rearrangement of genetic material, especially by crossing over in chromosomes or by the artificial joining of segments of DNA from different organisms.

**DNA ligation folowing by plasmid purification and bacteria transformation**

<img src=".\images\ligate.jpeg" alt="ligate" style="zoom:80%;" align="left"/>

<img src=".\images\hydrogen bond.png" alt="hydrogen bond" style="zoom:80%;" align="left"/>



**Yeast homologous recombination(ligation in vivo) folowing by plasmid isolation and bacteria transformation**

<img src=".\images\gap repair.jpg" alt="gap repair"  align="left"/>

<img src=".\images\mutagenic PCR.gif" alt="mutagenic PCR"  align="left" />

# Yeast plasmids

![yeast plasmids](.\images\yeast plasmids.png)

# Localization

![loca](.\images\loca-16368473641893.jpg)

# Positive or Negative charge

Double- and single-stranded DNA and RNA are all **strongly negatively charged**, with sub-nanometer inter-phosphate charge separations. This high charge density means that the free energy of nucleic acid (NA) interactions will be strongly influenced by electrostatics.

![DNAgel](.\images\DNAgel.jpg)

Proteins, however, are **not negatively charged**; thus, when researchers want to separate proteins using gel electrophoresis, they must first mix the proteins with a detergent called sodium dodecyl sulfate.

<img src=".\images\proteinGel.jpg" alt="proteinGel" style="zoom:150%;" />

# Percipetation

**It’s All About Solubility…**

First, we need to know why nucleic acids are soluble in water. Water is a polar molecule – it has a partial negative charge near the oxygen atom due to the unshared pairs of electrons, and partial positive charges near the hydrogen atoms. Because of these charges, polar molecules like DNA or RNA can interact electrostatically with the water molecules, allowing them to easily dissolve in water. Polar molecules can therefore be described as hydrophilic and non-polar molecules, which can’t easily interact with water molecules, are hydrophobic. Nucleic acids are hydrophilic due to the negatively charged phosphate (PO3–) groups along the sugar-phosphate backbone.

**The Role of Salt…**

OK, so back to the protocol. The role of salt in the protocol is to neutralize the charges on the sugar-phosphate backbone. A commonly used salt is sodium acetate. In solution, sodium acetate breaks up into Na+ and [CH3COO]–. The positively charged sodium ions neutralize the negative charge on the PO3– groups on the nucleic acids, making the molecule far less hydrophilic and, therefore, much less soluble in water.

**The Role of Ethanol…**

The electrostatic attraction between the Na+ ions in solution and the PO3– ions are dictated by [Coulomb’s Law](http://en.wikipedia.org/wiki/Coulomb's_law), which is affected by the dielectric constant of the solution. Water has a high dielectric constant, which makes it fairly difficult for the Na+ and PO3– to come together. Ethanol, on the other hand, has a much lower dielectric constant, making it much easier for Na+ to interact with the PO3–. This shields its charge and makes the nucleic acid less hydrophilic, thus causing it to drop out of the solution.

**The Role of Temperature…**

Incubation of the nucleic acid/salt/ethanol mixture at low temperatures (e.g. –20° or –80°C) is commonly cited as a necessary step in protocols. However, according to Maniatis *et al.* (Molecular Cloning, A Laboratory Manual 2nd Edition… 2nd edition?? – I need to get a newer version!), this is not required, as nucleic acids at concentrations as low as 20 ng/mL will precipitate at 0–4°C, so incubation for 15–30 minutes on ice is sufficient.

**The Wash Step With 70% Ethanol…**

This step is to wash any residual salt away from the pelleted DNA.

# growth temperature

**Bacteria**---It is not necessary that agar plates should be incubated at **37 degree** it totally depends on the species of microorganism you are trying to grow. **Generally pathogenic microorganisms are incubated at 37 degrees**, as 37 is normal body temperature of humans so the pathogens grow greatly at this temperature. E.coli which is widely studied microorganism is enterobacteriaceae family member with reside in intestinal tract so they require normal body temperature to grow.

Yeast---optimal growth temperature

<img src=".\images\yeastgrowthtemp.jpg" alt="yeastgrowthtemp" style="zoom:100%;" align="left"/>

# endogenous splicing

![image-20210828195520252](.\images\image-20210828195520252.png)

Picture left, measures splicing of all endogenous genes, this reflects the splicing(other introns don't have to have that pairing), right pic, this result explain because of the particular sequence in the intron, this reporter happens to have the sequence that will pair to this mutant of U6,therefore, it will get spliced better, cell on average. 

Completion of the budding **yeast genome** sequencing project helped to determine a total of **6,275 genes** on 16 chromosomes (12 million base pairs). Yeast possesses 23% homologous genes to humans; therefore, it is considered as a useful model for gene function studies. 

Approximately **5% of protein-coding genes in yeast contain introns**, and only nine contain more than one. By contrast, 90% of genes in mammals contain introns, with an average of eight introns per gene. In yeast, as in other organisms, introns have been viewed as the dispensable by-product of exon ligation because of their rapid degradation after splicing.

**A yeast gene usually have at most one intron.** The branch site is strongly conserved, whereas the polypirimidine tract is short.

**in *yeast* about 300 of the 6,275 genes contain introns, about 16 genes have multiple introns*.** 



# Molarity and Normality

You can convert from molarity (M) to normality (N) using the following equation:

**N = M\*n**

where n is the number of equivalents

To illustrate further, molarity and normality of some acids and bases are given below:

<img src="https://westlabblog.files.wordpress.com/2017/05/wl_article_molarity_diagram_02.jpg?w=450&h=300" alt="WL_ARTICLE_MOLARITY_DIAGRAM" style="zoom:100%;" align="left"/>



# Splicing

<img src=".\images\splicing.jpg" alt="splicing" style="zoom:100%;" align="left" />



<img src=".\images\group1.jpg" alt="group1" style="zoom:100%;" align="left" />

# Kinase & ATPase

<img src=".\images\KinaseATPase.png" alt="KinaseATPase" style="zoom:100%;" align="left" />

# Presentation

**catalytic center exists in two competing conformations** 

<img src=".\images\gr5.jpg" alt="gr5" style="zoom:120%;" align="left" />

To better understand if Cwc2, Prp8, U6 upstream of ACAGA motif are stabilizing the intron by making the clamp together, we used a genetic system in yeast. Modulation of four important factors involved in positioning of 5'SS, Cwc2, Prp8, U6 upstream of ACAGA motif, and the intron substrate, reveals their dynamic interplay at the stage of transition between the two catalytic steps of splicing.

Following the first catalytic step, Prp16 promotes the transition out of the catalytic conformation, which results in inhibition of first-step catalysis and entry into an intermediate state between the two catalytic steps. A defective allele of Prp16 (e.g., prp16-302), which fails to promote this transition, results in relative stabilization of the first-step catalytic conformation. This stabilization improves first-step catalysis but inhibits the overall progression of splicing. Previously, several mutant alleles of U6 upstream of ACAGA motif have been identified that counteract (suppress) defects of prp16-302 and thus **would be predicted to destabilize first-step catalytic interactions.** Here, Kasia tested Cwc2 and Prp8 mutants and analyzed their effects on prp16 alleles, which serve as genetic markers of the specific stage between first-step catalysis and transition out of the catalytic conformation. Similarly, we **employ suboptimal intron substrates defective at particular stages of splicing.** By testing relative effects of these various factors affecting defined stages of splicing, we can modulate and monitor in vivo dynamic changes that affect function of the catalytic center.

**if all these mutants in U6, Cwc2 and Prp8, destabilize the catalytic conformation by disrupting their joined interaction with intron downstream of the 5’SS.**



The effects of mutations U6-A45g and U46a can be explained by their consequences on base-pairing to intron positions +7 & +8;  the tested U6 alleles extend the region of complementarity, creating additional U6-A45g:5'SS-C8 or U46a:5'SS-U7 interactions and thus stabilizing binding of the 5'SS for catalysis. However, changes at U6 pos. 42ACA44 generally destabilize this binding and inhibit the first-step of splicing by facilitating Prp16 action, with U6-A42g and C43u alleles particularly strongly destabilizing the 5'SS positioning.These results confirm earlier studies (Staley and Guthrie, 1999) documenting the importance of intron pairing interactions with U6. Furthermore, they illustrate an unanticipated effect of the hyperstabilization of intron-U6 pairing at the 5'SS, resulting in inhibition of the second step of splicing. This effect suggests that at least transient destabilization of the intron-U6 contacts is needed for the transition from the first to the second step of splicing.

 **She used suboptimal substrates to test the effect on the splicing.**





<img src=".\images\image-20210906105934303.png" alt="image-20210906105934303" style="zoom:100%;" align="left" />

splicing is basically the process of introns are removed and exons are joined together. In spliceosome-catalyzed splicing of pre-mRNA process,

they coincide together in one region at the same time, 

we think Prp8,cwc2,U6 up stream are stabilizing the intron by making the clamp together, so we have three different components interact and influence the 5'SS stability. if All three compotents contribute to the same event, to the stabilizing, then inactivation of anyone of them will destabilize 5'SS. if it's really strongly unstable, then destabilizing another components will not make it much worse, because it's already unstable. 

made strongger mutant. 

because Cwc2 is known interact with closely with U6, we decided to look if Cwc2 has something to do with. we did screen and isolated number of mutants and they really behave in the same way. Cwc2 is also stabilize the first step catalytic center. mutants in Cwc2 made first step of conformation less effective, less solid, less functional.

about Prp2: some of the mutants that we isolated made it even worse. mutants that we isolated from the screen really effect the first step conformation. they destabilize the first step conformation.

mutants in Prp8 far from the clamp also destabilize the first step of conformation, maybe it doesn't bind well to the spliceosome,

we have mutants that destabilize first step of conformation, and yet they do not improve reporter splicing in our assay.  

1.we have three components of spliceosome, U6,Cwc2,Prp8,that in each of these factors, we have identified suppressors of Prp16-302. and we know in cryo-EM structure they are clustered together. 

2.is it enough to say, they really interact with eachother? is it enough to say they interact with eachother for stabilizing the 5'SS for catalysis? (for this we combine the different mutants together, to see what will happen when we combine them?) 

to linked to 5'SS stabilization, we have to find a mutant that would do the opposite.I could make them paired. so far we mutate and destabilize the interaction, because they were all suppressors of Prp16.that was how they were selected. this is why we take AT45/46ga mutant (this was a known mutatnt to stabilize the 5'SS) , now you combine them with something which does the opposite. this is for U6. you make this combine these mutants in one molecule in U6, **but you can also do that in the context of Cwc2 and Prp8 mutants**. but you take U6 that stabilizes with Prp8 mutants that destabilizes or Cwc2 mutants that destabilizes. 

we hopefully say that **Cwc2,Prp8 contact the same region and this clamp that stabilizes the 5'SS.** 

in the Cryo-EM structure there are close together and they don't serve to be functional. 



# slide

we have known the ACAGA motif in U6 stabilizing the 5'SS for the first step of catalysis, why is it important? what else can also added to  the stabilizing? 



Briefly, entry into the catalytic phase of splicing is facilitated by Prp2, resulting in the rearrangement of complex Bact to the activated complex B∗ , coincident with the formation of the first-step catalytic conformation. The first catalytic reaction generates complex C, which is subsequently remodeled by Prp16 to complex C∗ , marking the exit from the first-step catalytic conformation. Following second-step catalysis in complex P, the spliced mRNA product is released in the Prp22-dependent step (Schwer and Gross, 1998), and the resulting post-catalytic ILS complex is disassembled.

Following the first catalytic step, Prp16 promotes the transition out of the catalytic conformation, which results in inhibition of first-step catalysis and entry into an intermediate state between the two catalytic steps. A defective allele of Prp16 (e.g., *prp16*-*302*), which fails to promote this transition, results in relative stabilization of the first-step catalytic conformation. This stabilization improves first-step catalysis but inhibits the overall progression of splicing. Previously, several mutant alleles of U6 have been identified that counteract (suppress) defects of *prp16*-*302* and thus would be predicted to destabilize first-step catalytic interactions.  *prp16*-*R686I*, implying the generality of these effects.

# Practice

The main interest of those in our laboratory is the splicing mechanism. But the experiments we are doing now are mainly about the catalytic phase of splicing. The catalytic phase of splicing is a two-step process, the first step is the branching and the second step is the exon ligation. The ATPase Prp16 governs equilibrium between these two step conformations of the spliceosome. ATP hydrolysis by Prp16 can destabilize the branching conformation and result in transition to the exon ligation conformation. Prp16-302 impairs ATP hydrolysis at low temperatures and stalls Prp16 on the spliceosome right after branching. Therefore, whatever suppresses the prp16-302 defects is actually destabilizing the branching conformation. If we look at the cryo-EM structure, we can see that Cwc2,Prp8 and U6 upstream of ACAGA are located in one physical region next to the 5'SS of the intron. We hypothesized that these three things together would stabilize 5'SS.previous genetic screen of these three elements showed that several mutants of these suppress the defect of prp16-302. which means they destabilize the brancing conformation. now I am analysing the joined interaction of these three elements and it's effect on spliceosome confomation.



The spliceosome is a large, highly dynamic ribonucleoprotein complex made up of five small nuclear RNAs (U1, U2, U4, U5, and U6 small nuclear RNAs [snRNAs]) and over 100 proteins, which catalyzes pre-mRNA splicing by two consecutive transesterification reactions



If we stop Prp16 from working, the equilibrium will change, therefore the first step of spliceosome conformation will increase but the second step spliceosome conformation will decrease. in result the second step of splicing decrease and cell growth is inhibited. So if anything can make up for this deficiency and move from the first stage to the second stage, then this thing will have played a role in destabilizing the spliceosome in the first stage. If the mutant protein destabilizes the first stage syphilis, then the wild-type protein stabilizes the first stage syphilis. Genetic testing of several proteins has shown that several modified proteins can repair the deficiency of prp6. This means that these wild-type proteins can stabilize the syphilis in the first stage. If we look at the cryo-EM structure, we can see that Cwc2,Prp8 and U6 upstream of ACAGA are located in one physical region next to the 5'SS of the intron. We hypothesized that these three things together would stabilize 5'SS. I am currently working on whether or not the three things will destabilize the spliceosome conformation if the joint interaction with the intron are disrupted.

# 15/09/2021

splicing in yeast is mainly two step process, the first step is the branching and the second step is the exon ligation, in the first step 2'OH group of Adenosine of branch site is need to nucleophilically attacks the 5′ splice site (5′SS). therefore positioning of 5'SS is important for the catalysis. ACAGA motif of U6 stabilizing the 5'SS for the catalysis, if it's pairing with intron disrupted, the splicing will be inhibited. besides that, there are other factors effecting stabilization of 5'SS, they are upstream of ACAGA motif. Cwc2 and Prp8. these three factors stabilizing the 5'SS for the catalysis by making a clamp.  this is the actual cryo-EM structure. these three factors making a clamp to interact with intron and stabilize the 5'SS.

Prp16 is ATPase that destabilize the first step catalytic conformation therefore, facilitate the transition to the second step. on the other hand, the mutant form of prp16, Prp16-302 impairs ATP hydrolysis at low temperatures and stabilize the first step catalytic conformation result in inhibition of overall progression of splicing. genetic screens for mutants of Prp8,Cwc2 and upstream of U6-ACAGA motif showed that there are several mutants counteract defects of prp16-302. suggesting that they are destabilizing the interactions that is important for the first step catalytic conformation.Therefore, wild types of these three factors are actually stabilizing the intercations that are important for the first step catalytic conformations. the question is what interactions?

next the effects of U6 alleles on splicing were analyzed with suboptimal intron substrates defective at first, second and both steps of splicing. the result shows that all U6 counteractors of Prp16-302 defects inhibit splicing for BS-C. A42g, C43u improve splicing for A3c, inhibit splicing for G5a by contrast A45g, U46a inhibit splicing for A3c, but improve splicing for G5a. the result of AU45,46ga can be explained by base pairing with the intron at U6 position 45,46, thus stabilize binding of the 5'SS for the catalysis. therefore, we can speculate that A42g, C43u generally destabilize binding with intron.

then in similary way, the effects of several alleles in Prp8 and Cwc2 on splicing were analyzed. the mutants of Prp8 and Cwc2 behave in a same way with U6, suggesting that all of these Prp8, Cwc2, U6 upstream of ACAGA are interacting with intron and stabilize the 5'SS. but we need to prove it experimentally. since we know AT45,46ga stabilizing the 1st step catalytic conformation by pairing with the intron, therefore, if we combine suppressors of Prp16-302 defects with AT45,46, the suppressors should 

conclusion: they are the same type mutations but effects are different. since A42g and C43u effect differently to AT45,46ga, therefore, next step, we plan to test the combination of A42g and C43u 



if it stabilizes some endogenous introns critical for viability



